script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2_2.txt

```diff
@@ -42,12 +42,12 @@ Yumizuka stares at me motionlessly.
 "........"
 I wonder if Yumizuka is feeling sick as well.
 <Page12>
-After all this time, Yumizuka Satsuki is one of the central figures in our class.
+By now, Yumizuka Satsuki is one of the central figures in our class.
 She's cheerful, easy to get along with, hardworking, and it's clear that she cares more about those around her than herself.
 <Page13>
 I think most of the boys like her, and there are no bad rumors about her among the girls.
 Naturally, there is always a crowd of people around Yumizuka.
-She is the opposite of me, who is not very sociable, and is a well-liked mood-setter.
+She is a well-liked mood-setter, the opposite of me who is not very sociable.
```

Comment:
Still reads a little awkward, will come back to this and think about improvements